ui/src/app/cohort-review/detail-tab-table/detail-tab-table.component.tsx

```diff
@@ -0,0 +1,290 @@
+      }
+    }
+
+    getParticipantData() {
+      const {cdrVersionId, id, namespace} = this.props.workspace;
+      const {cid} = urlParamsStore.getValue();
+      const pageFilterRequest = {
+        page: 0,
+        pageSize: 10000,
+        sortOrder: SortOrder.Asc,
+        sortColumn: this.props.columns[0].name,
+        pageFilterType: this.props.filterType,
+        domain: this.props.domain,
+      } as PageFilterRequest;
+
+      cohortReviewApi().getParticipantData(
+        namespace,
+        id,
+        +cid,
+        +cdrVersionId,
+        this.props.participantId,
+        pageFilterRequest
+      ).then(response => {
+        response.items.forEach(item => {
+          if (this.props.domain === DomainType[DomainType.VITAL]
+            || this.props.domain === DomainType[DomainType.LAB]) {
+            item['itemTime'] = moment(item.itemDate, 'YYYY-MM-DD HH:mm Z').format('hh:mm a z');
+          }
+          item.itemDate = moment(item.itemDate).format('YYYY-MM-DD');
+        });
+        this.setState({
+          data: response.items,
+          loading: false,
+        });
+      });
+    }
+
+    onSort = (event: any) => {
+      this.setState({sortField: event.sortField, sortOrder: event.sortOrder});
+    }
+
+    columnSort = (sortField: string) => {
+      if (this.state.sortField === sortField) {
+        const sortOrder = this.state.sortOrder === 1 ? -1 : 1;
+        this.setState({sortOrder});
+      } else {
+        this.setState({sortField, sortOrder: 1});
+      }
+    }
+
+    onPage = (event: any) => {
+      this.setState({start: event.first});
+    }
+
+    render() {
+      const {data, loading, rows, start, sortField, sortOrder} = this.state;
+      let pageReportTemplate;
+      if (data !== null) {
+        const lastRowOfPage = (start + rows) > data.length
+          ? start + rows - (start + rows - data.length) : start + rows;
+        pageReportTemplate = (start + 1) + ' - ' + lastRowOfPage + ' of ' + data.length
+          + ' records ';
+      }
+      let paginatorTemplate = 'CurrentPageReport';
+      if (data && data.length > rows) {
+        paginatorTemplate += ' PrevPageLink PageLinks NextPageLink';
+      }
+
+      const columns = this.props.columns.map((col) => {
+        const asc = sortField === col.name && sortOrder === 1;
+        const desc = sortField === col.name && sortOrder === -1;
+        const header = <React.Fragment>
+          <span
+            onClick={() => this.columnSort(col.name)}
+            style={styles.columnHeader}>
+            {col.displayName}
+          </span>
+          {asc && <i className='pi pi-arrow-up' style={styles.sortIcon} />}
+          {desc && <i className='pi pi-arrow-down' style={styles.sortIcon} />}
+        </React.Fragment>;
+
+        return <Column
+          style={styles.tableBody}
+          bodyStyle={styles.columnBody}
+          key={col.name}
+          field={col.name}
+          header={header}
+          sortable={true} />;
```

Comment:
nit: In JSX, if a prop is specified with no value, it will pass `true`. So this could just say `sortable`